app/src/components/annotation/AnnotationLabel.tsx

```diff
@@ -55,6 +55,8 @@ const getAnnotationDisplayValue = (
         annotation.label ||
         "n/a"
       );
+    case "none":
```

Comment:
```suggestion
    case "none":
```

Can you make it so that you don't get an inconsistent padding?
<img width="557" alt="Screenshot 2025-03-24 at 9 34 41 AM" src="https://github.com/user-attachments/assets/59aaf895-6332-46c7-a315-158a454d100d" />